README.md

```diff
@@ -46,7 +47,7 @@ audio processing may require [lame](https://sourceforge.net/projects/lame/files/
 
 #### 1. Acquire the required repositories
 `git clone https://github.com/azerothcore/aowow.git aowow`  
-`git clone https://github.com/Sarjuuk/MPQExtractor.git MPQExtractor`
+`git clone https://github.com/Sarjuuk/MPQExtractor.git MPQExtractor`  
```

Comment:
ew

Reply:
? actually I am using the main repo of MPQExtractor but I will leave this
the two spaces in the end are to say in markdown "leave a newline after"

Reply:
I meant the blankspace you added XD

Reply:
But now it is merged kek